tests/automations/sign_with_wallet_missing_nonwitnessutxo_accept.json

```diff
@@ -0,0 +1,21 @@
+{
+  "version": 1,
+  "rules": [
+    {
+      "regexp": "Unverified|Update|or third party|[S]?pend from|Wallet name|Review|Amount|Address|Confirm|Fees",
+      "actions": [
+        ["button", 2, true],
```

Comment:
This will require changes to work with Stax :)

Reply:
Indeed, now I remember that that's why I left the PR as WIP before...